src/main/java/edu/cornell/kfs/coa/businessobject/CuAccountGlobal.java

```diff
@@ -290,7 +262,10 @@ private void updateAccountBasicFields(Account account) {
 		}
 
 		if (StringUtils.isNotBlank(accountRestrictedStatusCode)) {
-			updateAccountRestrictedStatusCodeForAccountBeingEdited(subFundGroup, accountRestrictedStatusCode, account);
+		    if (LOG.isDebugEnabled()) {
```

Comment:
Is there a reason why the code is checking if debug is enabled on this line, but not on line 147? Does the debug function within the Logger class check to see if it is enabled before writing to the log?

Reply:
Yes, 
The contents of a log statement are processed first, then shown if the applicable log level is achieved.  If a log statement only displays static values, then there is no need to surround it with an if statement.  However, if there is some logic being processed to create the log statement, it is a good idea to surround it with an if statement to avoid the processing except when necessary.